compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingAttrs.td

```diff
@@ -17,49 +17,53 @@ include "mlir/IR/EnumAttr.td"
 //===---------------------------------------------------------------------===//
 
 class IREEEncoding_Attr<string name, list<Trait> traits = []>
-  : AttrDef<IREEEncoding_Dialect, name, traits>;
+    : AttrDef<IREEEncoding_Dialect, name, traits>;
 
-class IREEEncoding_I32EnumAttr<string name, string summary, list<I32EnumAttrCase> cases>
+class IREEEncoding_I32EnumAttr<string name, string summary,
+                               list<I32EnumAttrCase> cases>
     : I32EnumAttr<name, summary, cases> {
   let cppNamespace = "::mlir::iree_compiler::IREE::Encoding";
   let genSpecializedAttr = 0;
 }
 
 class IREEEncoding_EnumAttr<EnumAttrInfo enumInfo, string name = "">
-  : EnumAttr<IREEEncoding_Dialect, enumInfo, name>;
+    : EnumAttr<IREEEncoding_Dialect, enumInfo, name>;
 
 // Enums for tagging operand operation in an EncodingAttr
 def MATMUL : I32EnumAttrCase<"matmul", 0>;
-def CONV   : I32EnumAttrCase<"conv", 1>;
-
-def EncodingOpType : IREEEncoding_I32EnumAttr<"EncodingOpType",
-    "Tracks the type of operation of the operand.", [
-      MATMUL,
-      CONV,
-    ]>;
-
-def EncodingOpTypeAttr:
-  IREEEncoding_EnumAttr<EncodingOpType, "optype">;
-
-
-def PackedStorageAttr : IREEEncoding_Attr<"PackedStorage"> {
+def CONV : I32EnumAttrCase<"conv", 1>;
+
+def EncodingOpType
+    : IREEEncoding_I32EnumAttr<"EncodingOpType",
+                               "Tracks the type of operation of the operand.", [
+                                 MATMUL,
+                                 CONV,
+                               ]>;
+
+def EncodingOpTypeAttr : IREEEncoding_EnumAttr<EncodingOpType, "optype">;
+
+def PackedStorageAttr
+    : IREEEncoding_Attr<"PackedStorage",
+                        [DeclareAttrInterfaceMethods<
+                            IREEEncoding_EncodingLayoutAttrInterface, [
+                              "calculateStorageSizeInBytes",
+                            ]>]> {
```

Comment:
I don't expect changes other than this part in this file. Can you revert other changes?